README.rst

```diff
@@ -326,6 +326,8 @@ Apache 2.0 - See `LICENSE`_ for more information.
 
 .. |build| image:: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python.svg?branch=master
    :target: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/1akrbcvnw8481fce/branch/master?svg=true
```

Comment:
I think you might want [this](https://www.appveyor.com/docs/status-badges/#badges-for-projects-with-public-repositories-on-github-and-bitbucket) instead.
`https://ci.appveyor.com/api/projects/status/github/googlecloudplatform/google-cloud-python`




Would there be any benefit to using https://shields.io?